charts/gafaelfawr/README.md

```diff
@@ -81,5 +81,3 @@ The Gafaelfawr authentication and authorization system
 | tolerations | list | `[]` | Tolerations for the Gafaelfawr frontend pod |
 | vaultSecretsPath | string | None, must be set | Path to the Vault secret (`secret/k8s_operator/<host>/gafaelfawr`, for example) |
 
-----------------------------------------------
-Autogenerated from chart metadata using [helm-docs v1.5.0](https://github.com/norwoodj/helm-docs/releases/v1.5.0)
```

Comment:
I don't really know why this got stripped out. I'm definitely using `helm-docs`, but it doesn't seem to have a `helm-docs --version` or anything equivalent, so I don't know what version I'm using.

Reply:
After a bit of sleuthing with the fancy new (to me, anyway) `go version -m $(which helm-docs)`, I think this is because I built from source (with `go get`); the helm-docs version variable is set as part of their central build and release process and if it's unset then this part of the template is not used.

Reply:
Yeah, I use the pre-built binary from their GitHub releases rather than building it myself. Not a problem; I don't really care if that string comes and goes.